misk-hibernate/src/main/kotlin/misk/hibernate/RealTransacter.kt

```diff
@@ -46,7 +47,8 @@ internal class RealTransacter private constructor(
   private val threadLatestSession: ThreadLocal<RealSession>,
   private val options: TransacterOptions,
   private val queryTracingListener: QueryTracingListener,
-  private val tracer: Tracer?
+  private val tracer: Tracer?,
+  private val tokenGenerator: TokenGenerator
```

Comment:
nit: Should nullable parameters go below required ones? that is, should we move TokenGenerator above Tracer?